sspl_test/rabbitmq/rabbitmq_egress_processor.py

```diff
@@ -213,7 +215,13 @@ def _read_config(self):
             self._site_id = self._conf_reader._get_value_with_default(
                 self.SYSTEM_INFORMATION, self.SITE_ID, '')
 
-
+            # Decrypt RabbitMQ Password
+            decryption_key = encryptor.gen_key(self._cluster_id, ServiceTypes.RABBITMQ.value)
+            try:
+                self._password = encryptor.decrypt(decryption_key, self._password.encode('ascii'), "RabbitMQegressProcessor")
+            except Exception as e:
+                self._password = "sspl4ever"

```

Comment:
It is not a default password anymore. Just raise the exception back. Then we would immediately know something went wrong.